app/views/school/welcome_wizard/la_funded_devices_you_can_order.html.erb

```diff
@@ -0,0 +1,47 @@
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+
+    <%= form_for @wizard, url: welcome_wizard_school_path(step: @wizard.step) do |f| %>
+      <%= f.hidden_field :step %>
+
+      <h1 class="govuk-heading-xl">
+        You can order a range of devices
+      </h1>
+
+      <p class="govuk-body">
+        We recommend checking with independent settings as soon as possible to find out which device type
+        and IT settings would be most suitable for each pupil.
+      </p>
+
+      <p class="govuk-body">
+        Depending on availability, you can choose from:
+      </p>
+
+      <h2 class="govuk-heading-s">
+        Standard Windows laptops
+      </h2>
+      <p class="govuk-body">
+        You'll be responsible for arranging and maintaining safeguarding measures, like content filtering.
+      </p>
+      <h2 class="govuk-heading-s">
+        DfE Restricted Windows laptops
+      </h2>
```

Comment:
This hard coded list needs to be swapped for the partial see https://github.com/DFE-Digital/get-help-with-tech/blob/main/app/views/school/welcome_wizard/devices_you_can_order.html.erb#L15

Otherwise updates there will not be reflected on this page